waypoint config crashes

I installed waypoint via the Fedora repository.

The commande `waypoint config` crashes:
```
$ waypoint config
panic: flag redefined: app

goroutine 1 [running]:
flag.(*FlagSet).Var(0xc0001584e0, 0x3ad3f60, 0xc000dac0e0, 0x341eea2, 0x3, 0x34900c8, 0x26)
        /usr/local/go/src/flag/flag.go:851 +0x4b8
github.com/hashicorp/waypoint/internal/pkg/flag.(*Set).Var(0xc000154700, 0x3ad3f60, 0xc000dac0e0, 0x341eea2, 0x3, 0x34900c8, 0x26)
        /home/circleci/project/waypoint/internal/pkg/flag/flag_var.go:79 +0x6f
github.com/hashicorp/waypoint/internal/pkg/flag.(*Set).VarFlag(0xc000154700, 0xc000bbaa00)
        /home/circleci/project/waypoint/internal/pkg/flag/flag_var.go:72 +0x1e1
github.com/hashicorp/waypoint/internal/pkg/flag.(*Set).StringVar(0xc000154700, 0xc000bba900)
        /home/circleci/project/waypoint/internal/pkg/flag/flag_string.go:33 +0x212
github.com/hashicorp/waypoint/internal/cli.(*ConfigGetCommand).Flags.func1(0xc000dce180)
        /home/circleci/project/waypoint/internal/cli/config_get.go:144 +0x18a
github.com/hashicorp/waypoint/internal/cli.(*baseCommand).flagSet(0xc00014a000, 0x0, 0xc000e9fb40, 0x41324b)
        /home/circleci/project/waypoint/internal/cli/base.go:415 +0x224
github.com/hashicorp/waypoint/internal/cli.(*ConfigGetCommand).Flags(0xc000d5dfc0, 0xc000e9fb98)
        /home/circleci/project/waypoint/internal/cli/config_get.go:130 +0x5a
github.com/hashicorp/waypoint/internal/cli.(*ConfigGetCommand).Help(0xc000d5dfc0, 0xc000dce150, 0x34314fa)
        /home/circleci/project/waypoint/internal/cli/config_get.go:171 +0x2f
github.com/mitchellh/cli.(*CLI).commandHelp(0xc00014a140, 0x3ab07c0, 0xc00000e020, 0x3b052e0, 0xc000d5d2e0)
        /go/pkg/mod/github.com/mitchellh/cli@v1.1.2/cli.go:575 +0x894
github.com/mitchellh/cli.(*CLI).Run(0xc00014a140, 0xc00014a140, 0xc000d5c000, 0xc000b822a0)
        /go/pkg/mod/github.com/mitchellh/cli@v1.1.2/cli.go:265 +0x227
github.com/hashicorp/waypoint/internal/cli.Main(0xc000cda9e0, 0x2, 0x2, 0x0)
        /home/circleci/project/waypoint/internal/cli/main.go:97 +0x4ee
main.main()
        /home/circleci/project/waypoint/cmd/waypoint/main.go:14 +0xa2
```

--- a/internal/cli/config_get.go
+++ b/internal/cli/config_get.go
@@ -5,10 +5,10 @@ import (
 	"fmt"
 	"os"
 
+	"github.com/hashicorp/waypoint-plugin-sdk/terminal"
 	"github.com/hashicorp/waypoint/internal/clierrors"
 	"github.com/hashicorp/waypoint/internal/pkg/flag"
 	pb "github.com/hashicorp/waypoint/internal/server/gen"
-	"github.com/hashicorp/waypoint-plugin-sdk/terminal"
 	"github.com/posener/complete"
 )
 
@@ -17,7 +17,6 @@ type ConfigGetCommand struct {
 
 	json bool
 	raw  bool
-	app  string
 }
 
 func (c *ConfigGetCommand) Run(args []string) int {
@@ -48,11 +47,11 @@ func (c *ConfigGetCommand) Run(args []string) int {
 		Scope:  &pb.ConfigGetRequest_Project{Project: c.project.Ref()},
 		Prefix: prefix,
 	}
-	if c.app != "" {
+	if c.flagApp != "" {
 		req.Scope = &pb.ConfigGetRequest_Application{
 			Application: &pb.Ref_Application{
 				Project:     c.project.Ref().Project,
-				Application: c.app,
+				Application: c.flagApp,
 			},
 		}
 	}
@@ -140,12 +139,6 @@ func (c *ConfigGetCommand) Flags() *flag.Sets {
 			Target: &c.raw,
 			Usage:  "Output the value for the named variable only (disables prefix matching)",
 		})
-
-		f.StringVar(&flag.StringVar{
-			Name:   "app",
-			Target: &c.app,
-			Usage:  "Scope the variables to a specific app.",
-		})
 	})
 }
 
@@ -165,8 +158,11 @@ func (c *ConfigGetCommand) Help() string {
 	return formatHelp(`
 Usage: waypoint config-get [prefix]
 
-  Retrieve and print all config variables previously configured that have the given prefix.
-	If no prefix is given, all variables are returned.
+  Retrieve and print all config variables previously configured that have
+  the given prefix. If no prefix is given, all variables are returned.
+
+  By specifying the "-app" flag you can look at config variables for
+  a specific application rather than the project.
 
 ` + c.Flags().Help())
 }

--- a/internal/cli/config_set.go
+++ b/internal/cli/config_set.go
@@ -5,17 +5,15 @@ import (
 	"os"
 	"strings"
 
+	"github.com/hashicorp/waypoint-plugin-sdk/terminal"
 	"github.com/hashicorp/waypoint/internal/clierrors"
 	"github.com/hashicorp/waypoint/internal/pkg/flag"
 	pb "github.com/hashicorp/waypoint/internal/server/gen"
-	"github.com/hashicorp/waypoint-plugin-sdk/terminal"
 	"github.com/posener/complete"
 )
 
 type ConfigSetCommand struct {
 	*baseCommand
-
-	app string
 }
 
 func (c *ConfigSetCommand) Run(args []string) int {
@@ -49,7 +47,7 @@ func (c *ConfigSetCommand) Run(args []string) int {
 			Value: arg[idx+1:],
 		}
 
-		if c.app == "" {
+		if c.flagApp == "" {
 			configVar.Scope = &pb.ConfigVar_Project{
 				Project: c.project.Ref(),
 			}
@@ -57,7 +55,7 @@ func (c *ConfigSetCommand) Run(args []string) int {
 			configVar.Scope = &pb.ConfigVar_Application{
 				Application: &pb.Ref_Application{
 					Project:     c.project.Ref().Project,
-					Application: c.app,
+					Application: c.flagApp,
 				},
 			}
 		}
@@ -75,14 +73,7 @@ func (c *ConfigSetCommand) Run(args []string) int {
 }
 
 func (c *ConfigSetCommand) Flags() *flag.Sets {
-	return c.flagSet(0, func(set *flag.Sets) {
-		f := set.NewSet("Command Options")
-		f.StringVar(&flag.StringVar{
-			Name:   "app",
-			Target: &c.app,
-			Usage:  "Scope the variables to a specific app.",
-		})
-	})
+	return c.flagSet(0, nil)
 }
 
 func (c *ConfigSetCommand) AutocompleteArgs() complete.Predictor {
@@ -101,7 +92,11 @@ func (c *ConfigSetCommand) Help() string {
 	return formatHelp(`
 Usage: waypoint config-set <name> <value>
 
-  Set a config variable that will be available to deployments as an environment variable.
+  Set a config variable that will be available to deployments as an
+  environment variable.
+
+  This will scope the variable to the entire project by default.
+  Specify the "-app" flag to set a config variable for a specific app.
 
 `)
 }

